frontend/services/entities/mdm.ts

```diff
@@ -324,13 +338,51 @@ const mdmService = {
     )}`;
     return sendRequest("DELETE", path);
   },
+
   getCommandResults: (
     command_uuid: string
   ): Promise<IGetMdmCommandResultsResponse> => {
     const { COMMANDS_RESULTS: MDM_COMMANDS_RESULTS } = endpoints;
     const url = `${MDM_COMMANDS_RESULTS}?command_uuid=${command_uuid}`;
     return sendRequest("GET", url);
   },
+
+  getSetupExperienceScript: (
+    teamId: number
+  ): Promise<IGetSetupExperienceScriptResponse> => {
+    const { MDM_SETUP_EXPERIENCE_SCRIPT } = endpoints;
+
+    const path = `${MDM_SETUP_EXPERIENCE_SCRIPT}?${buildQueryStringFromParams({
+      team_id: teamId,
+    })}`;
+
+    return new Promise((resolve, reject) => {
+      resolve(createMockSetupExperienceScript());
+    });
+
+    // return sendRequest("GET", path);
+  },
+
+  downloadSetupExperienceScript: (teamId: number) => {
+    const { MDM_SETUP_EXPERIENCE_SCRIPT } = endpoints;
+
+    const path = `${MDM_SETUP_EXPERIENCE_SCRIPT}?${buildQueryStringFromParams({
+      team_id: teamId,
+      alt: "media",
+    })}`;
+
+    return sendRequest("GET", path);
+  },
+
+  uploadSetupExperienceScript: (file: File, teamId: number) => {
```

Comment:
```suggestion
  uploadSetupExperienceScript: (file: File, teamId?: number) => {
```

On the backend, this also follows the same form as the normal scripts endpoint. For no team, team_id is omitted from the form data.

Reply:
thanks will update.